integration-cli/docker_cli_run_unix_test.go

```diff
@@ -299,7 +299,10 @@ func (s *DockerSuite) TestRunEchoStdoutWithMemoryLimit(c *check.C) {
 	}
 }
 
-// should run without memory swap
+// TestRunWithoutMemoryswapLimit should run without memory swap limit.
```

Comment:
I see that but test name is already telling you that, I see no point in having this comment